python/tests/__init__.py

```diff
@@ -113,6 +113,22 @@ def _levelorder_nodes(self, u, l, level):
         for c in self.children(u):
             self._levelorder_nodes(c, l, level + 1)
 
+    def _minlex_postorder_nodes(self, u, l):
+        l.extend(self._minlex_postorder_nodes_helper(u)[1])
+
+    def _minlex_postorder_nodes_helper(self, u):
```

Comment:
A quick docstring describing what this function returns would be helpful I think.